src/KMonad/Pullchain/Components/Dispatch.hs

```diff
@@ -95,8 +97,8 @@ pull d = do
   atomically ((Left <$> popRerun) `orElse` (Right <$> waitSTM a)) >>= \case
     -- If we take from the rerunBuf, put the running read-process back in place
     Left e' -> do
-      logDebug $ "\n" <> display (T.replicate 80 "-")
-              <> "\nRerunning event: " <> display e'
+      sepDebug
+      logDebug $ "\nRerunning event: " <> tshow e'
```

Comment:
> Minor problem: Logging for `Rerunning event`(?) needs to be corrected.

I reckon we probably need to call `textDisplay` here

Reply:
Spot on, fixed locally, will push soon.